apps/projects/app/index.html

```diff
@@ -2,10 +2,18 @@
 <html>
   <head>
     <title>Projects App</title>
+    <style>
+      #projects > div {
+          height: 100%;
+      }
+      div[class^="AppView__Content"] {
+          height: 100%;
+      }
+    </style>
```

Comment:
I warned you - no bed of roses. But still the best option, IMO.

Reply:
As far as hacks go, this one is respectably small. `#projects > div` doesn't worry me; I think that will work fine. But this second one – I doubt it will work in a production environment. Those class names that you hooked into, "_starts with_ `AppView__Content`", come from `styled-components`. The caveat: I believe they're only added in development mode. In production, the only class names are those mangled ones, like `bz2dbk-3`.

I'm not sure what to do about this.

Reply:
If that's the case, then we're back in square 1. But it seems to me that it's not. I just checked the mainnet Aragon governance app and it looks like this:
![appview](https://user-images.githubusercontent.com/16065447/61329141-28420500-a80c-11e9-8800-c7e4e48ca861.png)

Reply:
Ok. That's good. 

For now.

I fear we may be relying on an implementation detail of aragonUI, which may only result in bugs in a future production deploy.

aragonUI will receive a major upgrade soon; I wonder if it will give us a better way to hook into these elements. We can revisit this then, and leave your solution for now.

Reply:
Yeah. Like, if they're gonna change this implementation detail, they better also allow for a better way to access their elements.